plugins/cube/cube.cpp

```diff
@@ -397,7 +397,7 @@ class wayfire_cube : public wf::plugin_interface_t
     /* Mouse grab was released */
     void input_ungrabbed()
     {
-        wf::get_core().set_cursor("default");
```

Comment:
why do you need to unhide the cursor here? This will very likely corrupt the counter, because we haven't had a `hide` request before. I think it should still be a simple `set_cursor`

Reply:
Sorry, I mistakenly assumed this is in core.